requirements/edx/base.in

```diff
@@ -77,7 +77,7 @@ edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions==2.3.1
 edx-django-utils
 edx-drf-extensions
-edx-enterprise
+git+https://github.com/edx/edx-enterprise.git@bexline/fix_dependencies
```

Comment:
this was for testing purposes only, once that pr merges this can be returned to just the package